source/triggers/database-triggers.txt

```diff
@@ -16,41 +16,47 @@ Overview
 --------
 
 Database Triggers allow you to execute server-side logic whenever a
-document is added, updated, or removed in a
-linked {+atlas+} cluster.
-You can use database Triggers to implement complex data interactions,
-including updating information in one document when a related document
-changes or interacting with a :doc:`service </reference/services>` upon 
-the insertion of a new document.
-
-Database Triggers use MongoDB :manual:`change streams </changeStreams>` to
-listen for changes to documents in a collection and pass :ref:`database events
-<database-events>` to their associated Trigger function. Unlike SQL data
-triggers, which run on the database server, Atlas Triggers run on a serverless
-compute layer that scales independently of the database server.
-
-You can use a :manual:`$match </reference/operator/aggregation/match>`
-expression document to filter change events based on
-field values. For more information, see :manual:`Query on Embedded
-Documents </tutorial/query-embedded-documents>` in the MongoDB server
-manual.
+document is added, updated, or removed in a linked {+atlas+} cluster.
+Unlike SQL data triggers, which run on the database server, triggers run
+on a serverless compute layer that scales independently of the database
+server. Triggers automatically call :ref:`Atlas Functions <functions>`
+and can forward events to external handlers through AWS EventBridge.
```

Comment:
prob worth linking to the page https://www.mongodb.com/docs/atlas/app-services/triggers/examples/send-events-aws-eventbridge/ or the EventBridge docs here